packages/core/hooks/useOpenWithMenuItems/index.tsx

```diff
@@ -44,7 +50,7 @@ const UNSUPPORTED_APPS_HEADER = {
     itemType: ContextualMenuItemType.Header,
 };
 
-const APPS = (fileDetails?: FileDetail): Apps => ({
+const APPS = (fileDetails: FileDetail | undefined, openInCfe: (() => void) | undefined): Apps => ({
```

Comment:
Should this be some sort of options object? Is passing in the callback here okay?